ml-agents-envs/mlagents_envs/side_channel/utils.py

```diff
@@ -0,0 +1,68 @@
+import uuid
```

Comment:
Instead of functions, should we have a SideChannelManager like C# that holds the dictionary (or list)?

Reply:
We can, I can try to make a prototype